config/config_helpers.go

```diff
@@ -354,13 +358,28 @@ func findInParentFolders(params []string, include *IncludeConfig, terragruntOpti
 }
 
 // Return the relative path between the included Terragrunt configuration file and the current Terragrunt configuration
-// file
-func pathRelativeToInclude(include *IncludeConfig, terragruntOptions *options.TerragruntOptions) (string, error) {
-	if include == nil {
+// file. Name param is required and used to lookup the relevant import block when called in a child config with multiple
+// import blocks.
+func pathRelativeToInclude(params []string, trackInclude *TrackInclude, terragruntOptions *options.TerragruntOptions) (string, error) {
```

Comment:
Were you able to get all the relative path helpers to work with multiple `include`?

Reply:
Oh, also, does this support multiple _levels_ of `include`? I understand that `foo/terragrunt.hcl` can have two `include` blocks now... But what if `foo/terragrunt.hcl` has an `include` pointing to `bar/terragrunt.hcl`, and `bar/terragrunt.hcl` itself has another `include` pointing to `baz/terragrunt.hcl`. Does that work? And if so, do we detect cycles? Either way, we should document explicitly what is/isn't supported.

Reply:
> Were you able to get all the relative path helpers to work with multiple include?

Yes. They also now accept an optional parameter to indicate which include the relative helper should apply to, which is required when there are multiple include blocks.

> Oh, also, does this support multiple levels of include?

Not yet. This still limits to a single level. I updated the docs to clarify this: 9d3a702